src/glossary.rst

```diff
@@ -5357,6 +5357,15 @@ struct value
 :dp:`fls_YmZfW9kWlbIX`
 A :dt:`struct value` is a :t:`value` of a :t:`struct type`.
 
+.. _fls_P7920ALJisrH:
+
+structurally equal
+^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_glRZUKhmaWmP`
+A :dt:`type` is a :t:`structurally equal` when its :t:`values` can be compared
```

Comment:
The "a" requires a noun - a structurally equal what? Did you mean "structurally equal type", or "a type has structural equality when ..."?